libs/application/templates/university/src/fields/Overview/AcceptConfirmationModal/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { useMutation } from '@apollo/client'
+import { Application, DefaultEvents } from '@island.is/application/types'
+import { SUBMIT_APPLICATION } from '@island.is/application/graphql'
+import { Box, Button, Icon, ModalBase, Text } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { FC } from 'react'
+import { review } from '../../../lib/messages'
+import * as styles from './AcceptConfirmationModal.css'
+
+type AcceptConfirmationModalProps = {
+  visibility: boolean
+  setVisibility: (visibility: boolean) => void
+  application: Application
+  refetch?: () => void
+  reviewerNationalId?: string
+}
+
+export const AcceptConfirmationModal: FC<
```

Comment:
Is this something specific to this application or will it be used in other future ones?
I'm just wondering if there should be a `submitWithConfirmation` field in the shared lib?